crates/oxc_linter/src/snapshots/img_redundant_alt.snap

```diff
@@ -0,0 +1,189 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: img_redundant_alt
+---
+  ⚠ eslint-plugin-jsx-a11y(img-redundant-alt): Redundant alt attribute. Screen-readers already announce `img` tags as an image. You don’t need to use the words `image`, `photo,` or `picture` (or any
+  │ specified custom words) in the alt prop.
+   ╭─[img_redundant_alt.tsx:1:1]
+ 1 │ <img alt='Photo of friend.' />;
+   · ──────────────────────────────
```

Comment:
The `alt` attribute is the only interesting bit, shall we point the span to `alt` only? Or even better `alt` and `Photo`.

Reply:
your point makes sense, thanks ! [b1e5991](https://github.com/oxc-project/oxc/pull/1571/commits/b1e5991499e78829ad1d82c0adf35751ad32646e) 👍